docker-compose.test.py2.yml

```diff
@@ -196,12 +203,13 @@ services:
   localstack:
     image: localstack/localstack:latest
     ports:
-      - '4572:4572'
-      - '4566:4566'
+      - "4566-4599:4566-4599"
```

Comment:
Why you are exposing all ports? From what I remember s3 is on port 4572 only.

Reply:
As I can see you are using port 4566 as 4572 is actually deprecated. Then leave only 4566 open.

Reply:
Yes, exactly, I was looking for the reference because I've read about it too ;)